StPetersburg/InEtn66217.xml

```diff
@@ -0,0 +1,231 @@
+                           <dimensions type="outer" unit="mm">
+                              <height>175</height>
+                              <width>130</width>
+                           </dimensions>
+                        </extent>
+                        </supportDesc>
+                        <layoutDesc>
+                          <layout columns="2" writtenLines="19"></layout>
+                        </layoutDesc>
+                  </objectDesc>
+                  <handDesc>
+                     <handNote xml:id="h1" script="Ethiopic">
+                        <seg type="script"/>
+                        <seg type="ink">Black, red</seg>
+                        <desc/>
+                     </handNote>
+                  </handDesc>
+                  <decoDesc>
+                    <decoNote xml:id="d1" type="ornamentation">
+                      <locus target="#11r"></locus>
+                      <desc>Ornamental band with white, red and black interwoven lines.</desc>
+                    </decoNote>
+                    <decoNote xml:id="d2" type="ornamentation">
+                      <locus target="#49r"></locus>
+                      <desc>Ornamental band with black and white interwoven lines.</desc>
+                    </decoNote>
+                  </decoDesc>
+                  <additions>
+                    <list>
+                      <item xml:id="a1">
+                        <locus target="#7r"></locus>
+                        <desc type="GuestText">A table of shadows length during daytime in different months.</desc>
+                        <q xml:lang="gez"></q>
+                      </item>
+                      <item xml:id="a2">
+                        <locus target="#7v"></locus>
+                        <desc type="GuestText"><title type="incomplete" ref="LIT1834Mahlet"></title></desc>
+                        <note>18 lines of the text.</note>
+                        <q xml:id="gez"></q>
+                      </item>
+                      <item xml:id="a3">
+                        <locus from="63r" to="64r"></locus>
+                        <desc type="GuestText"></desc>
+                        <note>The quoted is the incipit.</note>
+                        <q xml:lang="gez">መንፈሶሙ፡ ነደ፡ እሳት፡ እሙንቱ፡ ወቅዱሳን፡ ኪሩቤል፡ ወሱራፌል፡ እለ፡ ፮ ክነፊሆሙ፡ ኪያከ፡ ይሴብሑ፡ በዓቢይ፡ ቃል፡ ወይዜምሩ። በቃል፡ ዘኢያርምም፡ ወይብሉ፡ ቅዱስ፡ ቅዱስ፡ ቅዱስ፡ እግዚአብሔር፡
+                        ፀባዖት፡</q>
+                      </item>
+                    <item xml:id="a4">
+                      <locus target="#64r'"></locus>
+                      <desc type="MagicText">Magic text against enemies with application instructions written in Amharic</desc>
+                      <note>The quoted is the incipit</note>
+                      <q xml:lang="gez">በርሃ፡ ይእቲ፡ ከሪ፡ ይሪ፡ ጠው፡ ራነ፡ በጀሊን፡ <gap reason="ellipsis"></gap> ለእለ፡ መጽኡ፡ እምሥራቅ፡ ወለእለ፡ መጽኡ፡ እምዕራብ፡ ለእለ፡ መጽኡ፡ እምሰሜን፡
```

Comment:
maybe add an empty <q xml:lang="am"/> to make this findable for anyone looking for Amharic additions?

Reply:
done